client/src/reducers/cartReducer.js

```diff
@@ -0,0 +1,39 @@
+import { CART_ACTIONS } from '../actions/types';
+import addItem from '../utils/cart/addItem';
+import removeItem from '../utils/cart/removeItem';
+import changeItem from '../utils/cart/changeItem';
+
+const initialState = {
+  size: 0,
+  opened: false,
+  loading: false,
+  items: []
```

Comment:
You put all these properties in our common store. This is no good, i feel like, wouldn't it be better to create property 'cart' to store all those properties in it?

Reply:
or at least rename it to 'cartOpened'